src/libs/collect.c

```diff
@@ -1508,6 +1508,40 @@ static void list_view(dt_lib_collect_rule_t *dr)
                    _("group leaders"),  _("group followers"), where_ext);
         break;
 
+      case DT_COLLECTION_PROP_MODULE: // module
+        snprintf(query, sizeof(query),
+                 "SELECT operation, 1, COUNT(*) AS count"
+                 " FROM main.images AS mi"
+                 " JOIN "
+                 "   (SELECT imgid, operation FROM main.history) h"
+                 "  ON h.imgid = mi.id"
+                 " WHERE %s"
+                 " GROUP BY operation"
```

Comment:
I'm wondering if group_by and order_by won't be slow without an index for history.operation. Have you seen speed issue ?

Reply:
I haven't seen any issue there (so far).

Reply:
So nothing to do, will see if needed later.